bin/workflows/pycbc_make_sbank_workflow

```diff
@@ -171,7 +171,6 @@ class CombineHDFBanksExecutable(wf.Executable):
 _desc = __doc__[1:]
 parser = argparse.ArgumentParser(description=_desc)
 pycbc.add_common_pycbc_options(parser)
-parser.add_argument('--version', action='version', version=__version__)
```

Comment:
Looks like you can also drop the definition of `__version__` above, and maybe one import?

Reply:
Yeah I saw that - it was the fact that all the `__author__` information was around it and I wasn't sure if that should be kept for some reason

Reply:
I doubt it, it might be a legacy from an old Python writing style.

Reply:
Okay, removed (and with the below examples as well)